### PR TITLE
[KernelGen][Nvidia] Add linalg_tensorsolve operator with Triton kernel

### PR DESCRIPTION
**Pull request by bwbwzzz**
## Summary
Adds a Triton kernel for `linalg_tensorsolve`. Triton kernel implementation for linalg_tensorsolve.

## Testing
- Parametrized tests over multiple shapes and float16/float32/bfloat16 dtypes (6 cases)
- Validated against PyTorch reference on device side
- Tested on: Nvidia

## Performance
Test command: `pytest benchmark/test_linalg_tensorsolve.py --level core` (NVIDIA H20)

### float16
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 4, 4 | 0.066 | 0.007 | 8.756 |
| 8, 8 | 0.070 | 0.008 | 9.017 |

### float32
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 16, 16 | 0.091 | 0.007 | 12.176 |
| 32, 32 | 0.113 | 0.008 | 14.618 |

### bfloat16
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 64, 64 | 0.166 | 0.008 | 20.871 |
| 128, 128 | 0.269 | 0.008 | 34.012 |

**Arithmetic Mean Speedup: 16.575x**
## Multi-backend Testing
| Backend | Accuracy Test | Benchmark | Speedup (mean) | Notes |
|---|---|---|---|---|
| Nvidia (H20) | PASS | PASS (6 cases, --level core) | 16.575 | Primary |
| Tianshu | PASS | PASS (6 cases) | 0.38 | — |
| Muxi | PASS | PASS (6 cases) | 0.92 | — |
| Ascend | PASS | — | — | — |
| Hygon | PASS | PASS (6 cases) | 0.45 | — |

## Files Changed
- `src/flag_gems/ops/linalg_tensorsolve.py`: Triton kernel implementation
- `tests/test_linalg_tensorsolve.py`: Accuracy test
- `benchmark/test_linalg_tensorsolve.py`: Performance benchmark
- `src/flag_gems/ops/__init__.py`: Register import and `__all__`
- `src/flag_gems/__init__.py`: Register to `_FULL_CONFIG`
- `conf/operators.yaml`: Add operator entry (kind: Math, stage: alpha 5.4)

### CODE REVIEW

File: conf/operators.yaml [hunk 1]
```diff
@@ -4265,6 +4265,17 @@ ops:
       - Math
     stages:
       - alpha: '5.4'
+  - id: linalg_tensorsolve
+    description: Triton kernel implementation for linalg_tensorsolve.
```
Comment (tengqm) [inline conf/operators.yaml:4269]: Please revise the description to be something meaningful.

File: conf/operators.yaml [hunk 2]
```diff
@@ -4265,6 +4265,17 @@ ops:
...
+    for:
+      - linalg_tensorsolve
+    labels:
+      - aten
+      - KernelGen
+    kind:
+      - Math
```
Comment (tengqm) [inline conf/operators.yaml:4276]: LinearAlg

File: tests/test_linalg_tensorsolve.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import pytest
+import torch
+
+import flag_gems
+
+from . import accuracy_utils as utils
+from . import conftest as cfg
+
+
```
Comment (tengqm) [inline tests/test_linalg_tensorsolve.py:9]: I believe we need to improve the test coverage given that there are many branches in the operator implementation.

**Comment by bin913** [comment]
@bwbwzzz Please fix failing unit-test
